packages/android_intent_plus/pubspec.yaml

```diff
@@ -4,6 +4,10 @@ version: 5.0.2
 homepage: https://plus.fluttercommunity.dev/
 repository: https://github.com/fluttercommunity/plus_plugins/tree/main/packages/android_intent_plus
 issue_tracker: https://github.com/fluttercommunity/plus_plugins/labels/android_intent_plus
+topics:
+  - android
+  - intent
```

Comment:
Topic for `intents` already exists which could be used as `intent` does not exist.
https://pub.dev/topics

Reply:
It is done on purpose. Android has `Intent` [mechanism](https://developer.android.com/reference/android/content/Intent), so I want to be specific that it is about it and not some generic intents.